### ISSUE TITLE
subquery

### ISSUE THREAD

**Issue by Rudolph-Miller** [fukamachi/sxql#12]
Is there the way to write
```
SELECT * from users
WHERE id NOT IN (
  SELECT id from banned_users
)
```
.

**Comment by Rudolph-Miller** [comment]
I found the way!
```
(select :*
  (from :users)
  (where
    (:not-in
      (list
        (select :id (from :banned_users))))))
```

**Comment by Rudolph-Miller** [comment]
Without `list` in `(:not-in`, it raised `type-error` of `SXQL-STATEMENT is not of type  (OR (AND #2=(SATISFIES TRIVIAL-TYPES:PROPER-LIST-P) CONS)
      (AND #2# NULL))`.

**Comment by fukamachi** [comment]
Fixed by #13. Closing.
